source/_components/arcam_fmj.markdown

```diff
@@ -0,0 +1,101 @@
+---
+layout: page
+title: "Arcam FMJ Receivers"
+description: "Instructions on how to integrate Arcam FMJ Receivers into Home Assistant."
+date: 2019-04-28 13:59 +0200
+sidebar: true
+comments: false
+sharing: true
+footer: true
+logo: arcam.svg
+ha_category: Media Player
+ha_release: 0.96
+redirect_from:
+  - /components/media_player.arcam_fmj/
+---
+
+The `arcam_fmj` platform allows you to control [Arcam FMJ Receveivers](https://www.arcam.co.uk/range/fmj.htm) from Home Assistant.
+
+Supported devices:
+
+- AVR 380 
+- AVR 450
+- AVR 750
+- Likely other AVRs
+
+
+To add a Yamaha Network Receiver to your installation, add the following to your `configuration.yaml` file:
+
+```yaml
+# Minimal example configuration.yaml entry
+arcam_fmj:
+  - host: hostname
+    zone:
+      1:
+```
+
+```yaml
+# Larger example configuration.yaml entry
```

Comment:
Move larger/more extensive examples to the bottom, below the configuration options table.